startup/GafferArnoldUI/arnoldExamples.py

```diff
@@ -0,0 +1,27 @@
+import Gaffer
+import GafferScene
+import GafferUI
+import GafferArnold
```

Comment:
This will break for any non-arnold-supporting builds. See other startup files for the `except ImportError` approach we take to that.

Reply:
So does GafferArnoldUI always load even if Arnold isn't supported in the build?

Reply:
If Arnold isn't supported in the build, then GafferArnoldUI won't exist. The approach we take is this :

- Everywhere except `startup/gui/menus.py`, we catch `ImportError` and silently fail.
- In `startup/gui/menus.py` we report errors rather than suppress them, so we only report the error in one place. We assume that if the user has gone to the trouble of putting `arnold` on PYTHONPATH, then they are expected to be able to use it, and any problem in loading GafferArnold should be reported.

This is slightly academic perhaps, because all public builds ship with Arnold support. But I want to maintain the possibility for a studio to maintain an internal build without Arnold support, instead using some other renderer.

Reply:
Sure thing, will add in, thanks - just thought that the `GafferArnoldUI` startup directory wouldn't be loaded unless `GafferArnoldUI` was initialising...

Reply:
Updated and pushed

Reply:
Sorry, wait, I'm talking nonsense again. You are right, GafferArnoldUI won't even exist in this case. I only recently [fixed that](https://github.com/GafferHQ/gaffer/commit/678d0a0ce3e26289642b6d96b0a305cfc0795fb8#diff-cc8093822b396591a5530a463c87d048), which I think is why I was getting confused.

Reply:
Un-updated and pushed :)